examples/GroceryExample/readme.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+The goal of this Grocery example is to try and instrument the code. From this
+excersize we hope to discover and learn additional topics for discussions.
+
+We are focus only on the API side at the moment.  It is known that SDK
+implementation will likely affect how the API is designed, but we will make our
+best judgement to tolerate the situation at this time.
+
+## Topics for discussions
+
+- Need some kind of concrete LabelSet() in API.  Access to LabelSetSdk() is
+unavailable from API side.
+
+- It is inconvenient to have to pass a default(SpanContext) when we don't care
+about spans. Need additional prototypes to make SpanContext optional.
```

Comment:
yes. maybe we can just make it default to Activity.Current, Baggage.Current.

Reply:
We have new APIs soon.  Let's address it then.